python_modules/libraries/dagster-dg/dagster_dg_tests/test_cache.py

```diff
@@ -68,7 +72,25 @@ def test_cache_no_invalidation_modified_pkg():
         assert "CACHE [hit]" in result.output
 
 
-def test_clear_cache():
+@pytest.mark.parametrize("clear_outside_project", [True, False])
+def test_clear_cache(clear_outside_project: bool):
+    with ProxyRunner.test(**cache_runner_args) as runner, example_project(runner):
+        result = runner.invoke("list", "component-type")
+        assert_runner_result(result)
+        assert "CACHE [miss]" in result.output
+        assert "CACHE [write]" in result.output
+
+        with pushd("..") if clear_outside_project else nullcontext():
+            result = runner.invoke("--clear-cache")
+            assert_runner_result(result)
+            assert "CACHE [clear-all]" in result.output
+
+        result = runner.invoke("list", "component-type")
+        assert_runner_result(result)
+        assert "CACHE [miss]" in result.output
+
+
+def test_clear_cache_outside_project():
```

Comment:
do you still need this test case with the clear_outside_project boolean in the test just above?

Reply:
good catch